app/scripts/controllers/detect-tokens.js

```diff
@@ -10,9 +10,13 @@ import {
   TOKEN_STANDARDS,
 } from '../../../shared/constants/transaction';
 import { EVENT, EVENT_NAMES } from '../../../shared/constants/metametrics';
+import { DETECT_TOKEN_ALARM } from '../../../shared/constants/alarms';
+import { isManifestV3 } from '../../../shared/modules/mv3.utils';
 
 // By default, poll every 3 minutes
 const DEFAULT_INTERVAL = MINUTE * 3;
+// FOR DEV PURPOSE -- NEED TO BE UPDATED TO 3 BEFORE MERGING
+const DEFAULT_INTERVAL_MV3 = 0.2;
```

Comment:
This interval will be updated to 3 before merging it to develop.
The chrome.alarms need to have an interval greater than or equal to 1 in production env but in development env we can use a value less than one to aid testing.